plugin/src/PatchTree.lua

```diff
@@ -147,6 +158,8 @@ function Tree:buildAncestryNodes(ancestry, patch, instanceMap)
     end
 end
 
+-- Builds a new tree from a patch and instanceMap
+-- uses changeListHeaders in node.changeList
 local function build(patch, instanceMap, changeListHeaders)
```

Comment:
If the function name was `buildTree` here, it would be obvious from the signature that it builds a new tree from a patch and instancemap without comments.

Reply:
I actually had that in previous commits but removed it cuz it was redundant- PatchTree.build() doesnt need to say its a tree twice.

Reply:
This looks like just a standalone build function instead of `PatchTree.build`, so it's not immediately obvious that it builds a tree or a component of a tree until you read how it's used.